.github/workflows/CI.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+on:
+  push:
+    branches:
+      - kokkos_develop_testing
```

Comment:
@sslattery, it will also trigger when you push on the `kokkos_develop_testing` branch in case we ever have to debug this. Or we could also add a manual trigger.

Reply:
Will this branch get automatically updated with the current state of `master`?